cmd/prometheus/main.go

```diff
@@ -58,6 +59,125 @@ func init() {
 	prometheus.MustRegister(version.NewCollector("prometheus"))
 }
 
+func newRootCmd() *cobra.Command {
+	rootCmd := &cobra.Command{
+		Use:   "prometheus",
+		Short: "./prometheus --config.file=\"prometheus.yaml\"",
```

Comment:
Prometheus is usually in the PATH, I'd not print the help text prefixed with `./`. Using backticks for the string will remove the need to escape the quotes. It would also be simpler to use show `prometheus --config.file=prometheus.yaml`.